### PR TITLE
WHAT IS FAST DATA page updated

### PR DESCRIPTION
**Pull request by crucio14**
## Description

I have updated the page by aligning it with the contents of https://mia-platform.eu/platform/fast-data/.

## Pull Request Type

-  [X] Documentation content changes
- [ ] Bugfix / Missing Redirects
- [ ] Docusaurus site code changes

## PR Checklist

- [X] The commit message follows our guidelines included in the [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md#how-to-submit-a-pr)
- [X] All tests of Lint, cspell and check-content pass. ([How to launch them?](../blob/main/CONTRIBUTING.md#content-checks))
- [X] No sensitive content has been committed

## Other information

I have added one image about DIH (taken from Figma). It's the same as in "Scale with the Digital Integration Hub" section of the https://mia-platform.eu/platform/fast-data/ (the image has

### CODE REVIEW

File: docs/fast_data/what_is_fast_data.md [hunk 1]
```diff
@@ -4,61 +4,132 @@ title: What is Fast Data
...
+3. Provide a declarative definition of Single Views, based on Projection fields and other flexible options.
+
+For further information about our No Code / Low Code logic, please visit the **[dedicated page](./no_code_overview.md)**.
+
+### Control Plane
+
+Control Plane is essential for monitoring the execution of Mia-Platform Fast Data and managing data across various runtime environments:
+
+* Monitor the status of various runtime pipelines.
+* Interact with users via a specialized frontend and adjust pipeline states in response to user inputs.
```
Comment (flower-of-the-bridges) [inline docs/fast_data/what_is_fast_data.md:96]:
```suggestion
* Using a dedicated frontend, users can resume or stop pipeline components by sending commands to the underlying microservices;
```

File: docs/fast_data/what_is_fast_data.md [hunk 2]
```diff
@@ -4,61 +4,132 @@ title: What is Fast Data
...
+Additionally, Mia-Platform Fast Data offers pre-built microservices and a user-friendly interface for setting up data flows in a low-code manner. This approach allows you to get started quickly and efficiently.
 
-Moreover, Fast Data provides ready-to-use microservices and a user-friendly interface to set up the data flow in a **low-code** fashion. The low-code configuration helps you to get started in a very short amount of time.
+Visit **[this page](./concepts/architecture.md)** to learn more about the architecture of Mia-Platform Fast Data.
 
-## How Fast Data can impact your business
+## Why Fast Data
 
-In this section we present the main scenarios in which Fast Data can significantly boost your company.
+Mia-Platform Fast Data combines **Data Mesh** and **Data Fabric** strategy, leveraging a suite of highly configurable tools that integrate the advantages of **Internal Developer Platform** while operationalizing your own **Digital Integration Hub**.
```
Comment (flower-of-the-bridges) [inline docs/fast_data/what_is_fast_data.md:31]: Here I would describe briefly what `Data Mesh` and `Data Fabric` strategies represents

**Review by flower-of-the-bridges — COMMENTED** [review]
Hi @crucio14 thanks a lot for your contribution! : ) 

I left two very small comments with small improvements

File: docs/fast_data/what_is_fast_data.md (hunk 2 above)
Comment (crucio14) [reply docs/fast_data/what_is_fast_data.md:31]: Hi Giovanni, below my proposals:

**Data Mesh** architecture is a decentralized alternative to central data lakes. It organizes data by specific business domains, giving domain teams ownership and accountability for their data, which enhances data quality and governance. Data Mesh allows teams to independently scale their data systems using the technologies and tools that best fit their needs.
**Data Fabric** is an adaptive, flexible, and secure integrated data architecture that provides a unified view of data across various sources, addressing challenges faced by data lakes. It simplifies data integration, transformation, and movement, allowing for faster and more efficient data access

Comment (flower-of-the-bridges) [reply docs/fast_data/what_is_fast_data.md:31]: Hi @crucio14 

Sounds good to me! :+1:

**Review by TheRusCompass — COMMENTED** [review]
Good job Luca! LGTM

File: docs/fast_data/what_is_fast_data.md [hunk 3]
```diff
@@ -4,61 +4,136 @@ title: What is Fast Data
...
+Mia-Platform Fast Data addresses these challenges by providing a solution where legacy systems are complemented by synced and transformed counterparts known as Projections, and aggregated versions stored in Single Views. These collections are maintained in MongoDB, which alleviates the need to repeatedly query and request for access to your DBs. Instead, the legacy systems are only required to send updates as events occur.
+
+### 24/7 operations
 
-### Centralization of data coming from different departments
+Mia-Platform Fast Data ensures an experience uninterrupted real-time data access, available 24/7, even during system maintenance or downtime. Our solution guarantees that data provisioning remains seamless and reliable, leveraging a resilient data layer that functions continuously regardless of API availability or system status. This setup enables you to serve channels and access critical information without disruption, ensuring that your data remains accessible and operational around the clock.
```
Comment (albertotessarotto) [inline docs/fast_data/what_is_fast_data.md:49]:
```suggestion
Mia-Platform Fast Data ensures uninterrupted access to real-time data, available 24/7, even during maintenance or system downtime. Our solution ensures that data provisioning remains seamless and reliable, leveraging a resilient data layer that operates continuously regardless of API availability or system status. This setup allows channels to be served and critical information to be accessed without interruption, ensuring that your data remains accessible and operational around the clock.
```

File: docs/fast_data/what_is_fast_data.md [hunk 4]
```diff
@@ -4,61 +4,136 @@ title: What is Fast Data
...
 
-If you are a medium or large company, you might have different departments each with their own applications and Systems of Record. Employees working in your back-office might need to access data that is saved in another system, but the application they are using is not allowed to access it.
-Thanks to Fast Data, you can centralize all the relevant data and make them available to the correct services, without exposing confidential information nor tinkering with complex access control systems.
+### Data standardization
 
-### Decreasing customer's wait time for aggregated data
+As mentioned before in Product overview section, Mia-Platform Fast Data gathers data from different System of Records (APIs, CDC, DB triggers etc.) and aggregates them into a single MongoDB collection called Single View and Projections, according to your business model.
```
Comment (albertotessarotto) [inline docs/fast_data/what_is_fast_data.md:53]:
```suggestion
As mentioned before in Product overview section, Mia-Platform Fast Data gathers data from different System of Records (APIs, CDC, DB triggers etc.) and aggregates them into a single MongoDB collection called Single View, according to your business model.
```

File: docs/fast_data/what_is_fast_data.md [hunk 5]
```diff
@@ -4,61 +4,136 @@ title: What is Fast Data
...
+A connector monitors changes in the data sources and forwards them to one or more Kafka topics. The data is then processed on parallel queues and converted into a format compatible with Mia-Platform Fast Data. This decouples data reading from processing, enhancing reliability and robustness.
+
+You can either develop a custom connector tailored to your specific needs or use one of the connectors available in the **[Mia-Platform Marketplace](/marketplace/overview_marketplace.md)**.
+
+### No Code / Low Code Data management
+
+Mia-Platform Fast Data implements its own data management logic that lets you save time during setup. We are talking about **No Code / Low Code (NC/LC)** functionalities.
+This system leverages two core principles: No Code Fast Data and Low Code Fast Data.
+
+With **No Code** functionality, you can configure your project quickly with minimal manual intervention. From version 10.6.0 onward, enabling the No Code feature (which may require coordination with your System Administrator) allows you to define relationships between projections within the ER Schema.
```
Comment (albertotessarotto) [inline docs/fast_data/what_is_fast_data.md:87]:
```suggestion
With **No Code** functionality, you can configure your project quickly with minimal manual intervention. From Mia-Platform Console v10.6.0 onward, enabling the No Code feature (which may require coordination with your System Administrator) allows you to define relationships between projections within the ER Schema.
```

File: docs/fast_data/what_is_fast_data.md [hunk 6]
```diff
@@ -4,61 +4,136 @@ title: What is Fast Data
...
+With **Low Code**, you only need to:
+1. Define relationships between Projections in the ER Schema.
+2. Specify which projection changes trigger updates to specific Single Views.
+3. Provide a declarative definition of Single Views, based on Projection fields and other flexible options.
+
+For further information about our No Code / Low Code logic, please visit the **[dedicated page](/fast_data/no_code_overview.md)**.
+
+### Control Plane
+
+Control Plane is essential for monitoring the execution of Mia-Platform Fast Data and managing data across various runtime environments:
```
Comment (albertotessarotto) [inline docs/fast_data/what_is_fast_data.md:97]:
```suggestion
Control Plane is essential for monitoring and governing the execution of Mia-Platform Fast Data pipelines across various runtime environments:
```

File: docs/fast_data/what_is_fast_data.md [hunk 7]
```diff
@@ -4,61 +4,136 @@ title: What is Fast Data
...
+
+For more information about Control Plane configuration, visit **[this page](/fast_data/runtime_management/control_plane.mdx)**. 
+
+## Fast Data to accomplish your Data Integration Hub
+
+Mia-Platform allows you to set up and deploy your end-to-end **Digital Integration Hub-enabled API platform**, as mentioned in the **[Gartner® Hype Cycle™ for Application Architecture and Integration, 2022](https://mia-platform.eu/news-en/gartner-application-architecture-and-integration/)**.
+
+![Mia-Platform Fast Data DIH architecture](img/fastdata-DIH-architecture.png)
+
+Digital Integration Hub is an important architectural component offering low-latency, high-throughput data access via APIs and event-based mechanisms. It separates SoRs sources from external channels through a data management layer that ensures data synchronization using event-driven patterns.
```
Comment (albertotessarotto) [inline docs/fast_data/what_is_fast_data.md:111]:
```suggestion
Digital Integration Hub is an important architectural component offering low-latency, high-throughput data access via APIs and event-based mechanisms. It separates SoRs sources from external channels through a data management layer that ensures data synchronization using event-driven patterns.

```

File: docs/fast_data/what_is_fast_data.md [hunk 8]
```diff
@@ -4,61 +4,136 @@ title: What is Fast Data
...
+
 ## How is Fast Data different from similar products?
 
-"ETL" systems (Extract, Transform, Load) are not big news, they have been around for decades and their principles are now a staple in the industry. So what makes Fast Data stand out from the crowd? It is easy to set up thanks to its **low code/ no code** functionality, fully customizable injecting actual javascript code as configuration, and part of an **ecosystem** that will grant you all the tools you need to make the most out of your standardized and aggregated data.
+"ETL" systems (Extract, Transform, Load) have been industry standards for decades, forming a core component of data management.
+What differentiates Mia-Platform Fast Data is its ease of setup through **No Code / Low Code** functionality, full customizability via JavaScript code as configured, and its integration within an **ecosystem** that provides all necessary tools for maximizing standardized and aggregated data.
+
+Moreover, you can build your Data Product utilizing a suite of tools specifically engineered to optimize Data acquisition and pipeline efficiency, for example:
```
Comment (albertotessarotto) [inline docs/fast_data/what_is_fast_data.md:127]: On the following bullet points, it could be useful to provide a link to each tool/feature listed below

**Comment by albertotessarotto** [comment]
Only few suggestion and a quick comment. Well done, thanks!

File: docs/fast_data/what_is_fast_data.md (hunk 8 above)
Comment (crucio14) [reply docs/fast_data/what_is_fast_data.md:128]: Hi Alberto, ok i provide a link to each tool/feature. Thanks!

**Review by fredmaggiowski — APPROVED** [review]
LGTM

**Pull request merged**